src/components/Requests/Request/Request.tsx

```diff
@@ -0,0 +1,58 @@
+import "./Request.scss";
+import {ReactComponent as HandIcon} from "assets/icon-hand.svg";
+import {useTranslation} from "react-i18next";
+import {UserAvatar} from "../../BoardUsers";
+import {Auth} from "../../../types/auth";
+
+type RequestType = "JOIN" | "RAISE_HAND";
+
+export type RequestProps = {
```

Comment:
no need for exporting, when only used in `Request.tsx`

```suggestion
type RequestProps = {
```